WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceView.swift

```diff
@@ -0,0 +1,117 @@
+import SwiftUI
+
+/// View for store performance on Dashboard screen
+///
+struct StorePerformanceView: View {
+    @ObservedObject private var viewModel: StorePerformanceViewModel
+    @State private var showingCustomRangePicker = false
+
+    init(viewModel: StorePerformanceViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        if viewModel.statsVersion == .v4 {
+            VStack(alignment: .leading) {
+                header
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                timeRangeBar
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                Divider()
+
+            }
+            .padding(.vertical, Layout.padding)
+            .background(Color(.listForeground(modal: false)))
+            .clipShape(RoundedRectangle(cornerSize: Layout.cornerSize))
+            .padding(.horizontal, Layout.padding)
+            .sheet(isPresented: $showingCustomRangePicker) {
+                RangedDatePicker(startDate: viewModel.startDateForCustomRange,
+                                 endDate: viewModel.endDateForCustomRange,
+                                 datesFormatter: DatesFormatter(),
+                                 customApplyButtonTitle: viewModel.buttonTitleForCustomRange,
+                                 datesSelected: { start, end in
+                    viewModel.didSelectTimeRange(.custom(from: start, to: end))
+                })
+            }
+        } else {
+            ViewControllerContainer(DeprecatedDashboardStatsViewController())
+        }
+    }
+}
+
+private extension StorePerformanceView {
+    var header: some View {
+        HStack(alignment: .top) {
```

Comment:
np:

![Screenshot 2024-04-16 at 19 59 29](https://github.com/woocommerce/woocommerce-ios/assets/266376/ce45b71f-5969-4d22-85f4-da836bee045e)

In the design, I think this should be vertically centered instead:
<img width="349" alt="Screenshot 2024-04-16 at 20 01 08" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/5a21663e-aa2a-40a4-b861-d6da7e47b135">

Reply:
Good catch! Updated in 7ba7f45f7680f308d518989bf95671839d70da27.